docs/asciidoc/handlers/ssl.adoc

```diff
@@ -85,6 +85,7 @@ import io.jooby.SSHandler
 
 For more information about SSL, please check the <<server-https-support, configure SSL>> section.
 
+// code review comment, this referenced doc doesn't exist
 [TIP]
 ====
 If you run behind a reverse proxy that has been configured to send the X-Forwarded-* header,
```

Comment:
There is a link to `link:proxy-peer-address.adoc[]` which doesn't exist, should the tip be removed or is there a more appropriate place to link to?

Reply:
it must be: <<router-trust-proxy, trust proxy>>